data/par/SPHERE+/sphere.py

```diff
@@ -119,7 +119,7 @@
 p_controller0.set_nwfs([0])         # /!\
 p_controller0.set_ndm([0, 1])       # /!\
 p_controller0.set_maxcond(1500)     #     determines the nb of modes to be filtered
-p_controller0.set_delay(1.15 * 1000 / 1380) # /!\ same delay in ms as in saxo.py
+p_controller0.set_delay(1.15 * 3000 / 1380) # /!\ same delay in ms as in saxo.py
```

Comment:
Did you check that `1.15 * 3000 / 1380` give the right delay in ms for the first stage ?

Reply:
Yes. the reason is that now apply_control is applied at every loop (ittime) or subframe, which is set to 1./3000 s. So in number of "subframes", the delay must be multiplied by 3 compared to what it was with the old version.

Reply:
Ok great thanks